app/views/vendors/vendor_products.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>
+  All Products:
+</h2>
+ <%= link_to "Add a New Product!", new_product_path(@vendors.id) %>
+  <ul>
+
+    <% @products.each do  |product| %>
+      <li>Product Name: <%= link_to product.name, product_path(@vendors.id, product.id) %> </li>
+      <li>Product Number: <%= product.product_number %> </li>
+
+      <%= button_to "delete", destroy_product_path(@vendors.id, product.id), { method: :delete, data: {confirm: 'Are you sure you want to delete?'}} %>
```

Comment:
Thank you! ^_^

Sent from my iPhone

> On May 3, 2016, at 3:11 PM, Jennie Buechner <notifications@github.com> wrote:
> 
> In app/views/vendors/vendor_products.html.erb:
> 
> > @@ -0,0 +1,15 @@
> > +<h2>
> > +  All Products:
> > +</h2>
> > + <%= link_to "Add a New Product!", new_product_path(@vendors.id) %>
> > +  <ul>
> > +
> > +    <% @products.each do  |product| %>
> > +      <li>Product Name: <%= link_to product.name, product_path(@vendors.id, product.id) %> </li>
> > +      <li>Product Number: <%= product.product_number %> </li>
> > +
> > +      <%= button_to "delete", destroy_product_path(@vendors.id, product.id), { method: :delete, data: {confirm: 'Are you sure you want to delete?'}} %>
> It's nice to have the user confirm before deleting from the database.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>